src/psycopmlutils/loaders/load_outcomes.py

```diff
@@ -38,3 +38,40 @@ def any_diabetes():
         msg.good("Finished loading any_diabetes event times")
         output = df[["dw_ek_borger", "timestamp", "value"]]
         return output.reset_index(drop=True)
+
+    @data_loaders.register("forced_admissions_indlagt")
+    def forced_admnissions_indlagt():
+
+        df = sql_load(
+            "SELECT dw_ek_borger, timestamp FROM [fct].[psycop_fa_outcomes_all_disorders_tvangsindlaeg_Indlagt_2y_0f_2015-2021]",
+            database="USR_PS_FORSK",
+            chunksize=None,
+        )
+
+        df = df[["dw_ek_borger", "datotid_slut", "six_month"]]
```

Comment:
If you want the `six_month` column, you need to select it in the SQL call as well :-)